caffe2/operators/quantized/int8_slice_op.h

```diff
@@ -35,6 +35,18 @@ class Int8SliceOp final : public SliceOp<CPUContext> {
         CAFFE_ENFORCE(HasArgument("ends"));
         CAFFE_ENFORCE_EQ(starts_.size(), ends_.size());
 
+        if (HasArgument("dim")) {
+          auto dim = this->template GetSingleArgument<int>("dim", 0);
```

Comment:
Nit: Check starts and ends don't exist, otherwise, at least print out some warning.

Reply:
Do you mean start_idx and end_idx? starts and ends are expected to exist for this op, I didn't change that behavior.

Reply:
No, I just meant starts and ends here. After latest changes, I think no need to check any more.